po/zh_CN.po

```diff
@@ -3386,19 +3389,21 @@ msgstr ""
 msgid ""
 "Unable to allocate %d MB * %d thread buffers; '%d: %s'. Please read ?fwrite "
 "for nThread, buffMB and verbose options."
-msgstr ""
+msgstr "无法分配 %d MB * %d 的线程缓存；'%d: %s'。请阅读 ?fwrite 中"
+"对 nThread、buffMB 和 verbose 选项的说明。"
 
 #: fwrite.c:822
 #, c-format
 msgid ""
 "Unable to allocate %d MB * %d thread compressed buffers; '%d: %s'. Please "
 "read ?fwrite for nThread, buffMB and verbose options."
-msgstr ""
+msgstr "无法分配 %d MB * %d 的线程压缩缓存；'%d: %s'。请"
+"阅读 ?fwrite 中对 nThread、buffMB 和 verbose 选项的说明。"
 
 #: fwrite.c:851 fwrite.c:883 fwrite.c:885
 #, c-format
 msgid "z_stream for data (%d): "
-msgstr ""
+msgstr "data （%d）的 z_stream："
```

Comment:
Double whitespace here :)

Reply:
it's actually just one :) but the `(` is a wide one from the chinese code range: `（`